tests/integration/Owncloud/OcisPhpSdk/OcisResourceTest.php

```diff
@@ -530,4 +530,61 @@ public function testResourcePreview(): void
             }
         }
     }
+
+    public function testFileGetResourcesMetadata(): void
+    {
+        $resourceMetadata = $this->personalDrive->getResourceMetadata('/somefile.txt');
+        $driveResources = $this->personalDrive->getResources();
+        foreach ($driveResources as $driveResource) {
+            if ($driveResource->getName() === $resourceMetadata['name']) {
```

Comment:
I think its better to make it explicit, to make sure there is not a parallel bug in both `getName` and `$resourceMetadata['name']`
```suggestion
            if ($driveResource->getName() === 'somefile.txt') {
```

same thing for the other tests